### ISSUE TITLE
Code should handle multiple tests and groups

### ISSUE THREAD

**Issue by arstrube** [tcmak/CppUTestEclipseJunoTestRunner#5]
Hi there,

```
* CppUTest Only supports matching Test Groups Names and/or Tests Names. 
* Thus this code ignore any input beyond testPaths[0]
```

Actually, CppUTest could have taken multiple entries all along; however, it would have processed them in a "last one wins" fashion.

Now that I am implementing proper handling of multiple tests and groups, we need the plugin to invoke CppUTest with complete information.

Multiple listings of the same test group won't do any harm, but ideally each test group would be listed only once. For any test to be run, both its group name and its test name must be present (empty test name or empty group name will always match).
